src/lib/main.js

```diff
@@ -194,6 +194,25 @@ module.exports = async (argv, env) => {
     await checkForUpdates({ nonFatal: true });
   }
 
+  let validationsFail = false;
```

Comment:
I think it'd be easier to read if this variable was positive instead of negative, eg:

```suggestion
  let validationsPassed = true;
```